src/components/article-description.astro

```diff
@@ -0,0 +1,10 @@
+---
+interface Props {
+  text: string;
+}
+const { text } = Astro.props;
+---
+
+<div>
+  <p>{text}</p>
```

Comment:
ok in this part is necessary to use a react component here you need to show how we can use the components, first of all you need to use the base layout . 
Example 
```
---
import Baselayout from '../../layouts/Layout.astro';
import ArticleTitle from '../../ui/components/atoms/article-title/ArticleTitle';
import ArticleDescription from '../../ui/components/atoms/article-description/ArticleDescription';

const uri = Astro.url.pathname;
---

<Baselayout title="Volumentes">
  <ArticleTitle title={uri} variant="primary" />
  <ArticleDescription article="This is the description of the page" />
</Baselayout>

```
Based in this you need to use these components, please refactor de code